engine/execution/ingestion/stop/stop_control.go

```diff
@@ -19,6 +21,11 @@ import (
 	"github.com/onflow/flow-go/storage"
 )
 
+const (
+	// TODO: figure out an appropriate graceful stop time (is 10 min. enough?)
+	MaxGracefulStopDuration = 10 * time.Minute
+)
```

Comment:
Maybe move `MaxGracefulStopDuration` to execution_config.go and unexport it (if it is only used there).

Alternatively, maybe (for consistency with existing code) use the number directly like other default settings in config.

My concern is client code might use this exported variable without realizing it is just the default.

Reply:
the code convention is to keep the constant within the module using it.  i'll rename it to DefaultMaxGracefulStopDuration to clarify this.